unlock-app/src/__tests__/hooks/useTermsOfService.test.ts

```diff
@@ -17,15 +16,15 @@ describe('useTermsOfService', () => {
 
   it('should return true if the correct value is set in localstorage', async () => {
     expect.assertions(1)
-    localStorage.setItem(`${APP_NAME}.${localStorageKey}`, 'true')
```

Comment:
I am confused because I think it is a good practice to namespace our stuff. why remove it?